src/mcp_server_docker/input_schemas.py

```diff
@@ -10,6 +10,35 @@
     field_validator,
     model_validator,
 )
+from pydantic.json_schema import GenerateJsonSchema, JsonSchemaValue
+from pydantic_core.core_schema import TupleSchema
+
+class CopilotAgentCompatibleSchema(GenerateJsonSchema):
```

Comment:
Can we name this class something a little more generic? Including a specific client name in the code base goes against the spirit of MCP a bit, in my opinion. Maybe something like:

```suggestion
class McpServerDockerJsonSchema(GenerateJsonSchema):
```